src/openforms/prefill/contrib/kvk/plugin.py

```diff
@@ -97,12 +93,9 @@ def check_config(self):
             # pass it on
             raise
         else:
-            if not isinstance(results, dict):
+            if not isinstance(result, dict):
                 raise InvalidPluginConfiguration(_("Response not a dictionary"))
-            items = results.get("resultaten")
-            if not items or not isinstance(items, list):
-                raise InvalidPluginConfiguration(_("Response does not contain results"))
-            num = items[0].get("kvkNummer", None)
+            num = result.get("kvkNummer", None)
             if num != check_kvk:
                 raise InvalidPluginConfiguration(
                     _("Did not find kvkNummer='{kvk}' in results").format(check_kvk)
```

Comment:
This translation literal is still not going correctly:

`python src/manage.py shell_plus`

```py
>>> from django.utils.translation import gettext as _
>>> check_kvk = "68750110"
>>> _("Did not find kvkNummer='{kvk}' in results").format(check_kvk)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    _("Did not find kvkNummer='{kvk}' in results").format(check_kvk)
KeyError: 'kvk'
```